server/server.go

```diff
@@ -236,18 +236,14 @@ func (s *server) Start(ctx context.Context, req *rpcpb.StartRequest) (*rpcpb.Sta
 	if *req.NumNodes < MinNodes {
 		return nil, ErrNotEnoughNodesForStart
 	}
-	customVMs := make(map[string][]byte)
-	if req.GetPluginDir() == "" {
-		if len(req.GetCustomVms()) > 0 {
+	if err := utils.CheckExecPath(req.GetExecPath()); err != nil {
+		return nil, err
+	}
+	chainSpecs := []blockchainSpec{}
+	if len(req.GetCustomVms()) > 0 {
```

Comment:
We have lost the `len(req.GetCustomVms()) == 0)` leg, does it matter?

Reply:
no. if was used only to check that plugin dir also was empty and ifnot return error. currently for the PR it is not the case, the plugin dir can be specified and will be saved to be used in future create blockchain calls